segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBCache.java

```diff
@@ -46,6 +47,31 @@
      */
     private static final int MAX_WRITE_AHEAD_LOG_SIZE_MB = 64;
 
+    /**
+     * Max number of in-memory write buffers (memtables) for the cache (active and immutable).
+     */
+    private static final int MAX_WRITE_BUFFER_NUMBER = 4;
+
+    /**
+     * Minimum number of in-memory write buffers (memtables) to be merged before flushing to storage.
+     */
+    private static final int MIN_WRITE_BUFFER_NUMBER_TO_MERGE = 2;
```

Comment:
What's the default for this parameter?

Reply:
Inspecting the code, `max_write_buffer_number = 2`.